### PR TITLE
Publish Audio Sharing

### PR DESCRIPTION
**Pull request by haecker-felix**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by haecker-felix** [comment]
bot, build de.haeckerfelix.AudioSharing

**Comment by haecker-felix** [comment]
bot, build de.haeckerfelix.AudioSharing

File: de.haeckerfelix.AudioSharing.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libadwaita.git",
+                    "tag" : "1.0.0-alpha.2"
+                }
+            ]
+        },
+        {
+            "name": "audio-sharing",
+            "buildsystem": "meson",
+            "run-tests": true,
```
Comment (A6GibKm) [inline de.haeckerfelix.AudioSharing.json:82]: Not necessary.

**Comment by haecker-felix** [comment]
bot, build de.haeckerfelix.AudioSharing

File: de.haeckerfelix.AudioSharing.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                    "url" : "https://github.com/lazka/sassc.git",
+                    "commit" : "82803377c33247265d779af034eceb5949e78354"
+                }
+            ]
+        },
+        {
+            "name" : "libadwaita",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dintrospection=enabled",
```
Comment (bilelmoussaoui) [inline de.haeckerfelix.AudioSharing.json:60]: Not needed

**Comment by haecker-felix** [comment]
Hm, for some reason `ninja dist` doesn't include the `vendor` folder anymore. No idea what's the cause, have to check it in detail tomorrow.

**Comment by barthalion** [comment]
Feel free to just upload vendor tarball separately if it helps with anything.

**Comment by haecker-felix** [comment]
bot, build de.haeckerfelix.AudioSharing

**Comment by haecker-felix** [comment]
Just tested the fresh 0.1.1 build, works fine :+1: 

image

**Pull request closed without merging**